src/index.html

```diff
@@ -9,11 +9,11 @@
     <div id="listcontainer">
       <h1>Today's To Do</h1>
       <form id="form" class="field">
-        <input type="text" id="addTask" placeholder="Add to your list...">
-        <button class="enterbutton" id="taskenter" type="button"><ion-icon name="arrow-back-outline"></ion-icon></button>
+        <input type="text" id="addTask" title="Add new Task" placeholder="Add to your list..." required>
+        <button class="enterbutton" id="taskenter" type="submit" title="Add new Task"><ion-icon name="arrow-back-outline"></ion-icon></button>
       </form>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add a **feature** that cleans the `input field` once a new file has been added. Right now, when a file has been added `the previous task text` remains in the `input_field`. Here is an **[example](https://stackoverflow.com/questions/4125247/how-does-form-reset-work)** or **[this](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset)**

- [ **OPTIONAL** ] For better organizing your project files. I suggest you create a directory named `modules/` then you can move there all your `modules_files`.